src/arm/bbai-bone-buses.dtsi

```diff
@@ -528,7 +528,7 @@ bone_eqep_2: &eqep1{
 
 // Bone sound
 &ocp {
-	bone_sound {
+	bone_sound: bone_sound {
```

Comment:
I'm going through these commit-by-commit so I can best understand what is going on. I will be looking for a future commit that merges these two files into one to know that the compatibility is working. It might have been nice to see changes to only one file, renaming it to a common-name and elimination of the other file.

Reply:
Bad comment. Didn't realize this *WAS* -bone-buses.dtsi.

Reply:
Note, please look at all places I2S could be routed out. I think thinking of this more as an I2S bus, rather than "sound" can be useful to making in generic. This is a new bus/interface like SPI, I2C, UART, etc. and should be treated the same in the cape compatibility documentation and definition.

Reply:
For example, I think it should likely be given an index.

Reply:
The I2S port is indexed, why not the audio port?